modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -683,6 +728,70 @@ private JSONArray _getItemsActionsJSONArray(
 			});
 	}
 
+	private String _getNestedFields(
+			String apiUrl, Set<ObjectEntry> fdsFieldsSet)
+		throws Exception {
+
+		if (fdsFieldsSet == null) {
+			return apiUrl;
+		}
+
+		String nestedFields = StringPool.BLANK;
+
+		int nestedFieldsDepth = 1;
+
+		JSONArray fdsFieldsJSONArray = JSONUtil.toJSONArray(
+			fdsFieldsSet,
+			(ObjectEntry objectEntry) -> {
+				Map<String, Object> properties = objectEntry.getProperties();
```

Comment:
@dsanz Finally, I cannot process all needed information in this lambda function (`nestedFieldDepth`) due to a `Declaration, final or effectively final variable expected` error :shrug: